tasks/Resources/Provisioning/Hive/provision.robot

```diff
@@ -36,7 +36,7 @@ Get Cluster Credentials
     Append to File  cluster_details.txt  password=${credentials_splited[1]}\n
 
 Login To Cluster
-    # Set Log Level    None
+    Set Log Level    None
```

Comment:
instead of using Set Log Level, you can register the keywords as "protected" in the RHOSi Setup. it manages to hide the values in logs without losing track of the keywords which are executed

Reply:
The problem is that I don't want to have a dependency on the tests. Do you think that we could move the protected keyword to a  task?  given that this has to be a temporary user I believe that there is no problem with deleting the log level at the moment